### ISSUE TITLE
~/.diffusionbee/downloads files should be placed in ~/Library/Caches/

### ISSUE THREAD

**Issue by alexbbrown** [divamgupta/diffusionbee-stable-diffusion-ui#183]
DiffusionBee downloads about 9GB of models

These are placed in a nonstandard location, making it hard for the user to discover and remove them when they are no longer wanted.

I suggest storing them in either

`~/Library/Application\ Support/DiffusionBee`

or better

`~/LibraryCaches/DiffusionBee`

which would allow the OS to manage them and kick them out

**Comment by Leland** [comment]
Very agreed. I vote caches. Also see #8. Few things to think about if this happens:

- Should we move existing users' downloads to the new folder? (I think yes?)
- Should that be requested via a prompt? (I think no, they're invisible caches – users won't care)
- Code should also consider if _both_ exist at the same time. What happens during migration then? (E.g. I run two versions of DiffusionBee)

> `~/LibraryCaches/DiffusionBee`

Small typo, only mention since paths matter: > `~/Library/Caches/DiffusionBee`

**Comment by enzyme69** [comment]
Just to add, I tried using different model like from the original SD Model, with Waifu Diffusion model, but somewhat still getting the same result (no change). 

Would be nice to be able to use custom model, and see the differences.

**Comment by zehrer** [comment]
Yes I just installed the new version and it download again the model.

**Comment by BrianLipke** [comment]
I can't for the life of me figure out the default location for the model files, do you know where they are?

**Comment by gingerbeardman** [comment]
> Yes I just installed the new version and it download again the model.

@zehrer this is normal behaviour if the local model is older version than the latest online model.

> I can't for the life of me figure out the default location for the model files, do you know where they are?

@BrianLipke they are at `~/.diffusionbee` (see #8)

**Comment by mattheys** [comment]
Can't this be configurable? I have all my stable diffusion stuff on an external SSD and I'd like to keep the models on that so I don't clog up my system drive.

**Comment by zehrer** [comment]
Yes please

**Comment by gingerbeardman** [comment]
Is there anything blocking this?
